recipe/meta.yaml

```diff
@@ -17,13 +17,19 @@ requirements:
     - python
     - pip
     - notebook >=4.4.1
```

Comment:
We shouldn't use `notebook` because it's not mentioned in the upstream source files. It was added 6 years ago in the conda-forge recipe. It was removed then from the upstream source install_requires https://github.com/jupyter-widgets/ipywidgets/commit/7ea681eb07e76796534ac1885d8e60ea9eb92b97#diff-e56eea957358c7c07362fde8ec3a3a8a8fbbeaa825b998cf4ea9191c55b1a0fd